src/assets/scripts/client/constants/themes/classic/modules/scope.js

```diff
@@ -15,6 +15,7 @@ export const SCOPE_THEME = {
     RUNWAY_LABELS: COLOR.WHITE_08,
     RUNWAY: COLOR.WHITE_04,
     SID: COLOR.BLUE_06,
+    STAR: COLOR.BLUE_06,
```

Comment:
How about `COLOR.RED` like the arrivals? Then, if both SIDs and STARs are shown, they can be distinguished (even if it's probably going to be just a hot mess of lines all over the place :wink:).

Reply:
done